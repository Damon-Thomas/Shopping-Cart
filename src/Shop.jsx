const Shop = () => {
    return (
      <div>
        <h1>Shop</h1>
        <p>working shop</p>
      </div>
    );
  };
  
  export default Shop;