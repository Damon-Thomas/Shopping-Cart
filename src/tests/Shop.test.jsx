import { render, screen } from '@testing-library/react';
import Shop from '../Shop';
import { expect } from 'vitest';




describe('Shop', () => {
  it('renders Shop as usual', () => {
    const { container } = render(<Shop />);
    expect(container).toMatchSnapshot();
  });
});





