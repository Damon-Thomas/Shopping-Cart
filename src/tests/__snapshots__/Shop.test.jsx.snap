// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Shop > renders Shop as usual 1`] = `
<div>
  <div
    class="shopContent"
  >
    <h1
      class="pageTitle"
    >
      Shop
    </h1>
    <div
      class="shopContainer"
    />
  </div>
</div>
`;
